library/options.inc.php

```diff
@@ -459,11 +459,13 @@ function generate_form_field($frow, $currvalue)
         if ($data_type == 4) {
             $modtmp = isOption($frow['edit_options'], 'F') === false ? 0 : 1;
             if (!$modtmp) {
+                $dateValue  = oeFormatShortDate(substr($currescaped, 0, 10));
                 echo "<input type='text' size='10' class='datepicker form-control' name='form_$field_id_esc' id='form_$field_id_esc'" .
-                " value='" . substr($currescaped, 0, 10) . "'";
+                " value='" .  attr($dateValue)  ."'";
             } else {
-                    echo "<input type='text' size='20' class='datetimepicker form-control' name='form_$field_id_esc' id='form_$field_id_esc'" .
-                    " value='" . substr($currescaped, 0, 20) . "'";
+                $dateValue  = oeFormatDateTime(substr($currescaped, 0, 20));
```

Comment:
TODO:
Add another "input-field" parameter to oeFormatDateTime that defaults to false. If true, then basically skip the Time formatting (so is HH:MM:SS). And then set it to true whenever populating a datetimepicker field.

Reply:
Actually, this already exists, just need to make the current parameter set to 0.

Reply:
DONE